hack/postgres-passwords-migration.sh

```diff
@@ -0,0 +1,30 @@
+#! /bin/bash
+# This script must be run to migrate all database passwords
+# before upgrading the synopsys-operator
+
+function usage(){
+  echo -e "usage:        ${0} [kubectl/oc] [NAMESPACE1] [NAMESPACE2] [...]\n"
+  echo -e "kubectl/oc:   Type which client you would like to use\n"
+  echo -e "NAMESPACE:    The blackduck namespace to migrate the secret from\n"
+}
+
+# check for number of arguments
+[ "$#" -lt 2 ] && echo -e "ERROR: At least one NAMESPACE must be provided.\n" && usage && exit
+
+# get the client; kubectl or oc
+client=$1
```

Comment:
Should check this is actually one of the approved options.  I could run "script NS NS" and still get past this with an invalid client.

Reply:
I tested it, and script `anything_invalid` `anything` will fail with `anything_invalid`: `command not found`, which I think is fine.  Not sure if we need anything more than that is required.  Script will also print `Using `anything_invalid` as the client, please make sure you have it on the machine you're running this script on`, which should make it clear to a user.

Reply:
The script will definitely fail with an invalid entry.  I'm suggesting adding a more clear error message.  Nothing is more confusing than an error message stating it can't find a binary you didn't think you were looking for. :)